bluemira/equilibria/run.py

```diff
@@ -477,6 +483,8 @@ def run_reference_equilibrium(self):
             relaxation=self.eq_config.relaxation,
             fixed_coils=True,
             diagnostic_plotting=self.eq_config.diagnostic_plotting,
+            keep_history=True,
```

Comment:
No point keeping history here it seems, the problem is not accessible a posteriori